app/javascript/.eslintrc.js

```diff
@@ -53,6 +53,14 @@ module.exports = {
     ],
     'react/jsx-no-target-blank': [2, { enforceDynamicLinks: 'always' }],
   },
+  overrides: [
```

Comment:
This ignores [barrel files](https://basarat.gitbook.io/typescript/main-1/barrel#:~:text=A%20barrel%20is%20a%20way,selected%20exports%20of%20other%20modules.) from this check.